src/Microsoft.AspNet.Diagnostics/Views/CompilationErrorPage.cs

```diff
@@ -193,7 +193,7 @@ public CompilationErrorPage()
 #line hidden
 
 #line 48 "CompilationErrorPage.cshtml"
-                         if (frame.Line != 0 && frame.ContextCode.Any())
+                         if (frame.Line != 0 && frame.ContextCode !=null && frame.ContextCode.Any())
```

Comment:
Maybe it's just a repo preference, in Razor we do the `?. == true` trick a lot. Not vastly important.